source/hands_on/sac.rst

```diff
@@ -270,7 +270,35 @@ Auto alpha strategy
             self._alpha_optim.step()
             self._alpha = self._log_alpha.detach().exp()
 
-The Benchmark result of SAC implemented in DI-engine is shown in `Benchmark <../feature/algorithm_overview.html>`_
```

Comment:
Comments:
1. In the Overview section, can we describe more about sac? For instance: What is sac, which categorisation does sac belong to? Why this algorithm is proposed and what's the final performance it achieves. A small trick would be to cite some sentences from the paper and rephrase them.

Reply:
1. 2. 3. 4. 5. Solved
6. To check later